builder.js

```diff
@@ -0,0 +1,36 @@
+class baseMessage {
+  setBold (val) { this.bold = val; return this }
+  setItalic (val) { this.italic = val; return this }
+  setUnderlined (val) { this.underlined = val; return this }
+  setStrikethrough (val) { this.strikethrough = val; return this }
+  setObfuscated (val) { this.obfuscated = val; return this }
+  setColor (val) { this.color = val; return this }
+  setInsertion (val) { this.insertion = val; return this }
+  setText (val) { this.text = val; return this }
+  setClickEvent (action, value) { this.clickEvent = { action, value }; return this }
+  setHoverEvent (action, type, id, name) {
+    this.hoverEvent = { action, contents: { type, id, name: { text: name } } }
+    return this
+  }
+}
+
+class Message extends baseMessage {
```

Comment:
add in a way to pass an existing json object to Message that gets merged into this